config/marklogic-sink-marklogic-ssl.properties

```diff
@@ -59,6 +58,9 @@ ml.dmsdk.threadCount=8
 # Optional - a comma-separated list of collections that each document should be written to
 ml.document.collections=kafka-data
 
+# Sets this to true so that the name of the topic that the connector reads from is added as a collection to each document inserted by the connector
+ml.document.addTopicToCollections=false
```

Comment:
Need to add this to config/marklogic-sink.properties as well, otherwise a null pointer is thrown.